librz/core/cfile.c

```diff
@@ -909,6 +909,7 @@ RZ_API RzCoreFile *rz_core_file_open(RzCore *r, const char *file, int flags, ut6
 	if (rz_io_is_listener(r->io)) {
 		rz_core_serve(r, fd);
 		rz_io_desc_free(fd);
+		fd = NULL;
```

Comment:
why is this necessary? It seems it's a dead assignment, because `fd`is not used in `beach`.

Reply:
Good catch! Removed that line.